### PR TITLE
Bug 1975696: compareOwnerReference should not accept a reference

### PR DESCRIPTION
**Pull request by glekner**
cc @spadgett @vojtechszocs

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/combined-disk.ts [hunk 1]
```diff
@@ -343,7 +340,7 @@ export class CombinedDiskFactory {
               (getOwnerReferences(p) || []).some((ownerReference) =>
                 compareOwnerReference(ownerReference, {
                   name: dataVolumeName,
-                  kind: kubevirtReferenceForModel(DataVolumeModel),
+                  kind: DataVolumeModel,
```
Comment (spadgett) [inline frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/combined-disk.ts:343]: Shouldn't this be...?

```suggestion
                  kind: DataVolumeModel.kind,
```

**Review by spadgett — CHANGES_REQUESTED** [review]

File: frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/combined-disk.ts (hunk 1 above)
Comment (glekner) [reply frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/combined-disk.ts:343]: oops

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**